.github/workflows/edge.yml

```diff
@@ -0,0 +1,64 @@
+name: Mondoo Docker Tag Edge
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Mondoo client version'     
+        required: true
+
+jobs:
+  build_container:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    name: Build Docker
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: arm64,arm
+
+      - name: Set up Docker BuildX
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          
+      - name: Build and push root images (alpine)
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/386,linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7
+          push: true
+          build-args: |
+            BASEURL=https://releases.mondoo.com/edge/mondoo/${VERSION}
```

Comment:
in a follow up I'd like to see this switched to install.mondoo.com

Reply:
this is a good idea but we will probably need to adjust the installer too since we need to be able to provide the version in the URL as the flow here is: tag -> use the tag version to pull the binary with the same version. I think the installer cannot do that now